genon/train/layout/change_filename.py

```
@@ -0,0 +1,27 @@
+with open(coco_path, "r", encoding="utf-8") as f:
+    coco = json.load(f)
+
+# 이미지 파일명 리스트 (정렬 보장)
+image_files = sorted(os.listdir(image_dir))
+
+# 이미지 수 = 어노테이션의 "images" 수인지 확인
+if len(image_files) != len(coco["images"]):
+    raise ValueError(f"🚨 이미지 수({len(image_files)})와 COCO 'images' 수({len(coco['images'])})가 다릅니다!")
+
+# 각 이미지 entry에 실제 파일명 부여
+for img_entry, fname in zip(coco["images"], image_files):
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)